front/src/containers/navegacion/Footer.js

```diff
@@ -1,28 +1,74 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { NavLink } from 'react-router-dom';
-import { Row } from 'reactstrap';
+import { Row, Badge } from 'reactstrap';
 import { Colxx } from '../../components/common/CustomBootstrap';
 import ModalEnviarInvitacion from 'containers/pages/ModalEnviarInvitacion';
+import ModalPlanes from 'containers/pages/ModalPlanes';
 import ROLES from 'constants/roles';
+import { getDocument } from 'helpers/Firebase-db';
 
 class Footer extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
       modalInvitacionOpen: false,
+      modalPlanesOpen: false,
+      docentes: 0,
+      alumnos: 0,
     };
   }
 
+  componentDidMount() {
+    this.getPlanInfo();
+  }
+
+  getPlanInfo = async () => {
+    let docentes = 0;
+    let alumnos = 0;
+    const institucion = this.props.institution.id;
+    const docObj = await getDocument(`usuariosPorInstitucion/${institucion}`);
+    const { data } = docObj;
+    for (const usuario of data.usuarios) {
```

Comment:
todo este for habría que encerrarlo en un if (data) así no rompe, en caso de que haya algun error al traer los datos. En caso de que no haya data, alumnos y docentes va a quedar en 0 y cuando se haga el setState va a mostrar el plan gratuito.

Reply:
Me parece bien! listo, hecho